website/docusaurus.config.js

```diff
@@ -0,0 +1,73 @@
+module.exports = {
```

Comment:
Please add the copyright header.

```
/**
 * Copyright (c) 2015-present, Facebook, Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
```

Reply:
Added!

Reply:
Sorry I wasn't clear, the copyright headers have to be added to all JS and CSS files. Basically any source code (except documentation) that's part of the website.

Reply:
There should be no problem now!